verdigado_template_default/views/res_partner.xml

```diff
@@ -6,9 +6,11 @@
         <field name="inherit_id" ref="base.view_partner_form" />
         <field name="arch" type="xml">
             <xpath expr="//group/group/field[@name='vat']" position="after">
+                <!--
                 <field name="x_gliederungsid" />
                 <field name="x_kunde_hosting" />
                 <field name="x_hauptdomain" />
+                /-->
```

Comment:
Ziel sollte sein, dass wir auf einer leeren Datenbank -i $alle_verdigado_module sagen können, oder? Dann mussten diese Felder zu "echten" Feldern migriert werden, soll ich ein migrations-script dafür schreiben und die Felder in `verdigado_partner` anlegen?

Reply:
Nee, andere Baustelle. Machen wir zu ner anderen Zeit.